index.ts

```diff
@@ -0,0 +1,50 @@
+import { test } from "bun:test";
+import { setupNetworks } from '@acala-network/chopsticks-testing'
+
+// run it with bun test ./index.ts --timeout=22000000
+// 22000000 miliseconds ~ 6.1 hours which should be enough (in theory) to run accounts migration
+test('test migration run', async() => {
+    const {polkadot, assetHub} = await setupNetworks({
+        polkadot: {
+            endpoint: 'wss://polkadot.rpc.permanence.io',
+            // resumes from the highest block.
+            resume: true,
+            // setupNetworks invokes dev.newBlock to override the wasm, so blocks in the db start from block+1
+            // block: 25172409, -- initial block
+            'wasm-override': 'runtime_wasm/polkadot_runtime.compact.compressed.wasm',
+            // 'runtime-log-level': 5,
+            'fetch-storages': '0x', // universal prefix
+            db: './rc-db.sqlite',
```

Comment:
```suggestion
            db: './dbs/polkadot.sqlite',
```

Let's stick to the chain name format I specified when I set up the repo, we'll be running this on multiple networks